src/components/form/question/part-1/1.jsx

```diff
@@ -1,7 +1,21 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import '../styleForm.css';
 
 const Question1 = () => {
+  const text = '';
```

Comment:
This variable is not used anymore, is it ? Please delete it and everything linked to it